components/src/dynamo/vllm/handlers.py

```
@@ -2796,26 +2796,24 @@ def _build_prompt_from_request(
 
         Returns:
-            Tuple of (prompt, error_dict) where:
-            - On success: (prompt, None)
-            - On failure: (None, error_dict to yield)
+            Tuple of (prompt, None). The second field preserves the existing
+            worker-handler contract.
+
+        Raises:
+            InvalidArgument: prompt embeddings are disabled or cannot be decoded.
         """
         if "prompt_embeds" in request and request["prompt_embeds"]:
```

Ruff (0.16.1)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)